djstripe/views.py

```diff
@@ -49,7 +49,7 @@ def get_context_data(self, *args, **kwargs):
         context['customer'] = customer
         try:
             context['subscription'] = customer.current_subscription
```

Comment:
Customer.current_subscription is now a property which is intended to work in a totally backwards-compatible fashion, unless the new setting DJSTRIPE_ALLOW_MULTIPLE_SUBSCRIPTIONS is turned on.